pkg/cloud/services/cloudformation/bootstrap.go

```diff
@@ -204,6 +204,7 @@ func controllersPolicy(accountID, partition string) *iam.PolicyDocument {
 					"elasticloadbalancing:DescribeTags",
 					"elasticloadbalancing:ModifyLoadBalancerAttributes",
 					"elasticloadbalancing:RegisterInstancesWithLoadBalancer",
+					"elasticloadbalancing:DeregisterInstancesFromLoadBalancer",
```

Comment:
IAM change will need a release note

Reply:
cc @ashoksekar07

Reply:
It was pointed out in today's community meeting to allow for a graceful fallback in case the permission is denied, which I think it's a valid point given that this is mostly a nice to have